tests/kernel/usb_drive.pm

```diff
@@ -49,8 +53,17 @@ sub run {
     assert_script_run "mount -t btrfs $device $mountpoint";
     assert_script_run "cd $mountpoint; md5sum -c $md5; cd /";
 
-    # cleanup
     assert_script_run("umount $mountpoint");
+
+    if (is_sle('16+')) {
+        assert_script_run "useradd -m -G users,disk unpriv -p $testapi::password";
+        zypper_call "in lklfuse";
```

Comment:
This should be `install_package('lklfuse');` so that we can test later SLE-16 service packs in transactional mode.

Reply:
good point, will change this.